spellchecker.py

```diff
@@ -0,0 +1,9 @@
+class SpellChecker:
+	"""SpellChecker class
+		-suggestion [string]: recommended sentence
+		-similarity [double]: similarity between input and recommendation
+	"""
+
+	def __init__(self, suggestion, similarity):
```

Comment:
This command is used for suggestions or words that were closer to the misspelled word